core/src/main/scala/org/apache/spark/sql/rapids/tool/store/StageModel.scala

```diff
@@ -54,7 +55,7 @@ class StageModel private(sInfo: StageInfo) {
     stubStage.submissionTime = newStageInfo.submissionTime
     stubStage.failureReason = newStageInfo.failureReason
     stubStage
-  }
+  }*/
```

Comment:
We can remove it completely
or just return the same `newStageInfo`. This will be more consistent with avoiding dead leaving dead code around.